rules/integrations/aws/exfiltration_rds_snapshot_restored.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2021/06/29"
+maturity = "production"
+updated_date = "2021/06/29"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when an attempt was made to restored RDS Snapshot. Snapshots are sometimes shared by threat actors in order to
+exfiltrate bulk data. If the permissions were modified, verify the snapshot was not shared with an
+unauthorized or unexpected AWS account.
+
```

Comment:
```suggestion
Identifies when an attempt was made to restored RDS Snapshot. Snapshots are sometimes shared by threat actors in order to
exfiltrate bulk data. If the permissions were modified, verify if the snapshot was shared with an
unauthorized or unexpected AWS account.
```

Reply:
Also, the more common exploitation path seems to be the restore of the RDS to access data, not the sharing